src/MemoryStorage.php

```diff
@@ -261,53 +292,314 @@ public function __construct(Kuzzle $kuzzle)
 
     public function __call($command, $arguments)
     {
-        if (!array_key_exists($command, $this->COMMANDS)) {
+        if (!isset($this->COMMANDS[$command])) {
```

Comment:
usage of `array_key_exists` is required, `isset` will throw an error if there are no `$command` key into `$this->COMMANDS` array